tests/test_visitors/test_filenames/test_module/test_module_redable_name.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from wemake_python_styleguide.violations.naming import (  # noqa: I001
+    UnreadableNameViolation,
+)
+from wemake_python_styleguide.visitors.filenames.module import (
+    WrongModuleNameVisitor,
+)
+
+
+@pytest.mark.parametrize('filename', [
+    'still1name',
+    'l1module',
+    'the1interstep',
+])
+def test_unreadable_filename(assert_errors, filename, default_options):
+    """Testing that unreadable characters combinations do not allowed."""
+    visitor = WrongModuleNameVisitor(default_options, filename=filename)
+    visitor.run()
+
+    assert_errors(visitor, [UnreadableNameViolation])
+
+
+@pytest.mark.parametrize('filename', [
+    'ordinary',
+    'first_module',
```

Comment:
We should also test `TestO_0`: is it valid?

Reply:
`TestO_0` has WrongModuleNamePatternViolation and UnderscoredNumberNameViolation. Why we should check it here if our unreadable list it:
```'lI',
    'Il',
    '1l',
    'l1',
    '1I',
    '1i',
    '0O',
    'O0'

Reply:
Because it is a corner case that is related to underscore name patterns.